### DISCUSSION TITLE
Powerbook 160 strange issue !

### DISCUSSION THREAD

**Discussion by Wawavoun** [erichelgeson/BlueSCSI-v1#149] [Q&A]
Hello,

I have replaced the internal scsi drive of a Powerbook 160 with a Bluescsi.

A have a SD memory with 3 active images that works very well into a standard Bluescsi (the one for scsi flat cable) connected to the Powerbook scsi expansion port.

If I put this SD memory into the internal Bluescsi the boot become hazardous.
Sometime (often after a reset with the rear switch) the Powerbook book and then everything seems to okay.
Sometime (generally after power on) the Powerbook wont boot.
We can see he try each image and finally play the 4 notes dead sound.

If I disable 2 of the 3 images just leaving a bootable one then the Powerbook boot well everytime ! So I am salvaged...

But it could be handy to work with multiple images. Any idea will be appreciated !

Regards.
Philippe

**Comment by erichelgeson** [comment]
> If I disable 2 of the 3 images just leaving a bootable one then the Powerbook boot well everytime ! So I am salvaged...

This sounds like a SCSI Driver issue. I would open all 3 images up in [Disk Jockey ](https://bluescsi.onegeekarmy.eu/diskjockey/) and validate they all have the same driver. 

If that is not it you might want to recheck your hardware to see if there is an issue - currently this repo does not have any powerbook boards.

**Reply by Wawavoun** [reply]
For sure they dont have the same scsi driver.
Two are like this :
[HD60_512_7.0.1_FR_NEW.hda.zip](https://github.com/erichelgeson/BlueSCSI/files/9175382/HD60_512_7.0.1_FR_NEW.hda.zip)
made with softmac especially for a Mac Plus. One is bootable and one is empty.
The third is the MacPack found here : https://archive.org/details/macpack-20210619  
If its a driver issue why is this working outside the Powerbook and not inside ?

**Reply by erichelgeson** [reply]
I dont know what hardware you're using inside the powerbook, that could be an issue, but defiantly a driver issue would cause this.

**Reply by Wawavoun** [reply]
It is this one : https://github.com/xunker/bluescsi_pb#powerbook-adapter-for-bluescsi

**Comment by Wawavoun** [comment]
I think more to a timing problem at boot. I assume the boot time of the bluescsi is linked to the number of stored images...
After a reset the timing will not be a problem because the bluescsi is ready to answer.
This is not the case at power up.

**Reply by erichelgeson** [reply]
The BlueSCSI parsing images takes a fraction of a second, I can't see how that would affect it. Even if it was "late" BlueSCSI doesnt respond to SCSI messages until it had read all files/config/etc.

**Comment by Wawavoun** [comment]
Hi,

I have try various images with different scsi drivers and the behaviour is always the same.
As soon as I put more than one image on the sd board the Mac play the 4 notes death chime at power up.
Then, after a reset with the rear switch, he boot well.

Regards.
Philippe
